### ISSUE TITLE
XFCE panel remains unthemed

### ISSUE THREAD

**Issue by swidi** [B00merang-Project/Windows-XP#4]
![img](https://i.imgur.com/wV2sCFu.png)

Amazing theme, but sadly the panel is not correctly themed. It has the window background color instead of the correct background image and the window buttons are themed like normal buttons

**Comment by Elbullazul** [comment]
will work on it when I get time to do it

**Comment by s3boun3t** [comment]
yes me too on xubuntu 18.04
please fix it
https://reho.st/self/b001450e905ff6c18dcaf14927193cf3c3f439fe.png

thanks

**Comment by Elbullazul** [comment]
since all of b00merang's themes are affected, we'll work on a general fix soon

**Comment by s3boun3t** [comment]
thanks for the quick answers

**Comment by mason1920** [comment]
The panel still remains unthemed.

**Comment by xmbv** [comment]
Is this project still being maintained? This issue still remains after over a year.

**Comment by Telavus** [comment]
Hi, this is still not fixed :( Any update on it? I saw latest commit was 5 days ago and I hope you didnt forget this

**Comment by Elbullazul** [comment]
I can fix it this week but may not be prefect in the first try.  At least i have the code to theme the panel correctly

**Comment by Elbullazul** [comment]
e78dcb6 adds full support: 

![VirtualBox_Xubuntu 19 04_16_06_2019_11_54_06](https://user-images.githubusercontent.com/15310985/59566372-7efdc880-902d-11e9-8bc1-08bb35c23891.png)

**Comment by Telavus** [comment]
Perfect! Just one final question for you :)

Do you have any possible remedy for the issue you can see on my panel with the white ugly coloring? 

https://imgur.com/hnYLBfg.png

**Comment by Elbullazul** [comment]
@Telavus what Xfce version are you on and on what distro? (name and version)

**Comment by Telavus** [comment]
@Elbullazul Manjaro (Rolling release) XFCE Version 4.13

**Comment by Telavus** [comment]
Mind you: The Luna theme seems to be working fine with color schemes, its specifically the Embedded and Royale / Royale Noir themes

**Comment by demonidios** [comment]
Elbullazul,  The Luna theme in XFCE 4.12/4.14(XUbuntu 19.04/19.10/20.04) has an incorrect appearance in the tray and does not match the appearance of the original Windows XP theme. If you give an email, then I can send a more detailed description of the observed inconsistencies and discrepancies.

**Comment by Elbullazul** [comment]
You can post it here. Please include screenshots as well

**Comment by demonidios** [comment]
![Elbullazul_1](https://user-images.githubusercontent.com/60550766/74270058-45dc3c00-4d13-11ea-980f-78966d67ba07.png)

No. 1,2,3 - the color does not match. The original Windows XP “Luna” theme has shades of blue, and you have light gray everywhere.
Circled in green. The “Start” button (in Russian - “Pusk”) - in the original theme it is green, dimensions - height - 30 pixels, length - 107 pixels. If necessary, I can send images in png format of the right size.
Circled in red. The icon, when you click on it, you get to the window with the buttons for rebooting, shutting down, sleeping and changing users - you have it green with white arrows, but should be red, as in the original theme.

**Comment by demonidios** [comment]
![image](https://user-images.githubusercontent.com/60550766/74270298-a8cdd300-4d13-11ea-9321-1f6034f24659.png)

The “Start” button (in Russian - “Pusk”) - in the original theme it is green, dimensions - height - 30 pixels, length - 107 pixels. When you hover over the cursor from dark blue, it becomes light blue. Must be green.
In the theme “Redmond97” (the theme for Windows 98), the theme developer has made the appearance settings of the “Start” button into a separate file called whisker-menu.css. In the gtk.css file, it is loaded through the following command: “@import url (" whisker-menu.css "); ". 
Download, look, maybe something will come in handy from what he applied for his topic.
Link - https://github.com/matthewmx86/Redmond97

**Comment by demonidios** [comment]
![image](https://user-images.githubusercontent.com/60550766/74270388-d0bd3680-4d13-11ea-8361-0fd5f1cec1f2.png)

Discrepancy of appearance of tray. It is possible that it will be more precisely called Taskbar Notification Area or System tray. In your theme, tray, like the entire panel, is dark blue. When you hover over any indicator, its background takes on a color of light blue, while the strips of dark blue remain above and below. In the original “Luna” theme, all indicators (prome of the keyboard layout indicator) in the tray are located against a light blue background, as in the image, and when you hover over them, only a message (tooltip) pops up, the indicator does not stand out anymore. The keyboard layout indicator remains on a dark blue background, when you hover over it, it is highlighted in the form of a square of the same color protruding above the surface of the panel (maybe I didn’t put it that way, so I’ll apologize for a possible misunderstanding. If that, pay attention to what the “Start” button looks like in the topic “Redmond97).

**Comment by demonidios** [comment]
I would like the corrected topic to be posted for use as quickly as possible, but I understand that you have more important matters. Christian, and for help on what issues related to the XUbuntu operating system can I contact you?

**Comment by demonidios** [comment]
I could fix the problem with the icons on the button myself.
To change the icon of the "Exit" button in the xfce-menu application menu, you need to log as root in /usr/share/icons/Windows-XP-master/24x24/actions/ and name the system-log-out file with the desired icon.
The icon of the reboot button changes to /usr/share/icons/Windows-XP-master/32x32/actions/ by adding a file with the desired image and assigning it the desired name.

**Comment by Elbullazul** [comment]
Hello,

Thanks for the redmond link,might be useful. 

I can't push your icon change because it's the action that is different. In xp you had a shutdown button. I xfce you have a logout button.

Panel theming for xfce is a pain because there is no fixed layout or guideline that I know of to theme it like other panels (cinnamon, gnome). Maybe I find something that helps in the redmond repo?

I'm currently employed full time and may not be able to work on this for a couple of months. However, I'll try my best to find some time to at least try to fix it before then.

Regards,

**Comment by demonidios** [comment]
Windows XP Icons pack
https://www.gnome-look.org/p/1215836/
https://github.com/Elbullazul/WinXP-Icons/archive/master.zip  :-)

**Comment by demonidios** [comment]
![Elbullazul_2](https://user-images.githubusercontent.com/60550766/74314568-4a8a0a00-4d7e-11ea-8205-f09b7e163dfa.png)

**Comment by demonidios** [comment]
![Elbullazul_3](https://user-images.githubusercontent.com/60550766/74314641-71e0d700-4d7e-11ea-937b-49a0dda154a6.png)

**Comment by demonidios** [comment]
All this is the same for XUbuntu 19.04 / 19.10 / 20.04

**Comment by demonidios** [comment]
Look here. Then the theme creator was able to figure out the Whisker-Menu button and the appearance of the panel and the menu itself.
Prowler EX WhiskerMenu and Panel theme GTK3 XFCE 4.14
https://www.xfce-look.org/p/1355567/

**Comment by Elbullazul** [comment]
tried some things, but results weren't satisfying enough for me. menu background is feasible, but I can't have it scale and there's no apparent zone separation for me to theme portions of the menu.

the start menu fix looked decent, but may not be released until next month

**Comment by demonidios** [comment]
Hello Christian! Could you make the necessary changes? In the last message from 04/23/2020 you write that you are implementing a new design theme with corrections no earlier than the next month. Is it already available for download and use, or do you still have to wait?

**Comment by Elbullazul** [comment]
@demonidios it's available in the master branch, haven't made a release yet

**Comment by demonidios** [comment]
Downloaded from the master branch. Nothing has changed. Either you forgot to add changes, or you just didn’t succeed.

**Comment by Elbullazul** [comment]
my bad, didn't remember what changes you asked for.

I'll try to have the app menu button look like in Windows XP, but the tray area, I'm afraid I can't do that yet. At least, not until Xfce finishes migrating to GTK3

**Comment by Elbullazul** [comment]
Also, I'm going to open a new issue for this matter, please use that one for further discussion (I'll post the link)

**Comment by Elbullazul** [comment]
#26
